src/pages/claim-page.tsx

```diff
@@ -1,11 +1,53 @@
-import {NextPage} from "next";
+import { NextPage } from "next";
 import ClaimView from "../components/Claim/ClaimView";
-import {serverSideTranslations} from "next-i18next/serverSideTranslations";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { NextSeo } from 'next-seo';
+import Head from "next/head";
+import { useTranslation } from "next-i18next";
 const parser = require('accept-language-parser');
 
 const ClaimPage: NextPage<{ personality, claim, href }> = ({ personality, claim, href }) => {
+    const { t } = useTranslation();
+    
+    let sentence;
+    let classification;
+    claim.content.object.forEach((paragraph) => {
+        paragraph.content.forEach((content) => {
+            if(content.props.topClassification) {
+                sentence = content.content;
+                classification = content.props.topClassification.classification
+            }
+        })
+    })
+
+    
     return (
-        <ClaimView personality={personality} claim={claim} href={href}/>
+        <>
+            <Head>
+                <script type="application/ld+json">{`
```

Comment:
Nice! But let's do this on a separate PR and use another issue for it. I have a lot of considerations for this feature.